lib/signs/realtime.ex

```diff
@@ -30,9 +30,7 @@ defmodule Signs.Realtime do
               [
                 :headway_stop_id,
                 :current_time_fn,
-                announced_arrivals: [],
                 announced_approachings: [],
-                announced_approachings_with_crowding: [],
```

Comment:
What was this used for?

Reply:
This was to record whether we had included a crowding message in the approaching announcement, and if not, we'd include it in the arriving message. Now that arriving is gone, this isn't relevant (doubly so, since crowding is disabled anyway)